pkg/testcase/certificates.go

```diff
@@ -0,0 +1,122 @@
+package testcase
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+)
+
+func errCheck(error error, message string, ignore bool) {
+	if error != nil {
+		shared.LogLevel("error", message)
```

Comment:
we dont need that!   also if you want we already have a shared func that logs and return err ! 
https://github.com/rancher/distros-test-framework/blob/dbc0224ca28dba1bd9567c33a0a89112ed2904bf/shared/aux.go#L290

Reply:
removed and using ReturnLogError instead